HW1/Task2.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+print('Введите трёхзначное число:')
+a = int(input())
```

Comment:
Вам не нужна функция `print` вы можете написать внутри функции `input` сообщения о вводе.
```suggestion
a = int(input('Введите трёхзначное число: '))
```